py/mock.py

```diff
@@ -26,7 +26,8 @@
        mock [options] {--init|--clean|
        --scrub=[all,chroot,cache,root-cache,c-cache,yum-cache,dnf-cache,lvm]}
        mock [options] [--rebuild] /path/to/srpm(s)
-       mock [options] --buildsrpm {--spec /path/to/spec --sources /path/to/src|--scm-enable [--scm-option key=value]}
+       mock [options] --buildsrpm {--spec /path/to/spec 
+       --sources /path/to/src|--scm-enable [--scm-option key=value]}
```

Comment:
Again, this does not look desirable as it will also affect line lengths of output.